### PR TITLE
Created a new Async task that creates zip for images

### LINKED ISSUE #2242: Adding Zip option for selected files in an album
**Actual Behaviour**
Presently the user can create a zip file only for a whole album but not for some selected files of an album

**Expected Behaviour**
The user should be able to create Zip file for selected files in an album

**Steps to reproduce it**
1) Open an album and select some files.
2) Click on the menu icon in the top-right corner.
3) No zip option is displayed.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by codepoet2017390**
Fixed #2242 

Changes: Created a new Async Task that creates a zip file for selected Images and saves in a folder called Phimpme Zips.

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java [hunk 1]
```diff
@@ -3496,6 +3512,95 @@ public void run() {
...
+
+            if(!albumsMode && !all_photos && !fav_photos){
+                getAlbum().clearSelectedPhotos();
+            }
+            else if(!albumsMode && all_photos && !fav_photos){
+                clearSelectedPhotos();
+            }
+            swipeRefreshLayout.setRefreshing(false);
+        }
+        }
```
Comment (abishekvashok) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: Please properly intend here.

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java [hunk 2]
```diff
@@ -3496,6 +3512,95 @@ public void run() {
...
+                e.printStackTrace();
+            }
+            return date;
+        }
+
+        @Override
+        protected void onPostExecute(String date) {
+            super.onPostExecute(date);
+            NotificationHandler.actionPassed(R.string.zip_completion);
+            String path = "ZIP: "+date+".zip";
```
Comment (abishekvashok) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: This meaning I cant create two zips on the same day?

Comment (codepoet2017390) [reply app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: @abishekvashok  oh i'm sorry i had used the date for testing purpose but didn't remember before pushing the code. I will add time too

**Comment by codepoet2017390** [comment]
@abishekvashok  updated please review

**Comment by abishekvashok** [comment]
Thanks :)

**Pull request merged**